13 - Slide in on Scroll/index-START.html

```diff
@@ -1,123 +1,321 @@
+        cupiditate hic quidem ex, quas, amet ipsum esse dolore facilis beatae
+        maxime qui inventore, iste? Maiores dignissimos dolore culpa debitis
+        voluptatem harum, excepturi enim reiciendis, tempora ab ipsam illum
+        aspernatur quasi qui porro saepe iure sunt eligendi tenetur quaerat
+        ducimus quas sequi omnis aperiam suscipit! Molestiae obcaecati officiis
+        quo, ratione eveniet, provident pariatur. Veniam quasi expedita
+        distinctio, itaque molestiae sequi, dolorum nisi repellendus quia
+        facilis iusto dignissimos nam? Tenetur fugit quos autem nihil,
+        perspiciatis expedita enim tempore, alias ab maiores quis necessitatibus
+        distinctio molestias eum, quidem. Delectus impedit quidem laborum, fugit
+        vel neque quo, ipsam, quasi aspernatur quas odio nihil? Veniam amet
+        reiciendis blanditiis quis reprehenderit repudiandae neque, ab ducimus,
+        odit excepturi voluptate saepe ipsam. Voluptatem eum error voluptas
+        porro officiis, amet! Molestias, fugit, ut! Tempore non magnam, amet,
+        facere ducimus accusantium eos veritatis neque.
+      </p>
+    </div>
 
-    <p>laborum neque repudiandae quam, cumque, voluptate veritatis itaque, placeat veniam ad nisi. Expedita, laborum reprehenderit ratione soluta velit natus, odit mollitia. Corporis rerum minima fugiat in nostrum. Assumenda natus cupiditate hic quidem ex, quas, amet ipsum esse dolore facilis beatae maxime qui inventore, iste? Maiores dignissimos dolore culpa debitis voluptatem harum, excepturi enim reiciendis, tempora ab ipsam illum aspernatur quasi qui porro saepe iure sunt eligendi tenetur quaerat ducimus quas sequi omnis aperiam suscipit! Molestiae obcaecati officiis quo, ratione eveniet, provident pariatur. Veniam quasi expedita distinctio, itaque molestiae sequi, dolorum nisi repellendus quia facilis iusto dignissimos nam? Tenetur fugit quos autem nihil, perspiciatis expedita enim tempore, alias ab maiores quis necessitatibus distinctio molestias eum, quidem. Delectus impedit quidem laborum, fugit vel neque quo, ipsam, quasi aspernatur quas odio nihil? Veniam amet reiciendis blanditiis quis reprehenderit repudiandae neque, ab ducimus, odit excepturi voluptate saepe ipsam. Voluptatem eum error voluptas porro officiis, amet! Molestias, fugit, ut! Tempore non magnam, amet, facere ducimus accusantium eos veritatis neque.</p>
+    <script>
+      function debounce(func, wait = 20, immediate = true) {
+        var timeout;
+        return function () {
+          var context = this,
+            args = arguments;
+          var later = function () {
+            timeout = null;
+            if (!immediate) func.apply(context, args);
+          };
+          var callNow = immediate && !timeout;
+          clearTimeout(timeout);
+          timeout = setTimeout(later, wait);
+          if (callNow) func.apply(context, args);
+        };
+        alert("Hi!");
```

Comment:
Please remove the alert method. Also, please take note that the codes after the return statement are not reachable. Therefore it will not run in the browser.